packages/smooth_ui_library/lib/animations/smooth_reveal_animation.dart

```diff
@@ -32,19 +32,24 @@ class _SmoothRevealAnimationState extends State<SmoothRevealAnimation>
         vsync: this,
         duration: Duration(milliseconds: widget.animationDuration));
     final CurvedAnimation curve = CurvedAnimation(
-        curve: widget.animationCurve, parent: _animationController);
+        curve: widget.animationCurve, parent: _animationController!);
     _animationOffset =
         Tween<Offset>(begin: widget.startOffset, end: Offset.zero)
             .animate(curve);
 
     Timer(Duration(milliseconds: widget.delay), () {
```

Comment:
Would it make any sense to `cancel()` the timer in `dispose()` instead of or in addition to setting `_animationController` to `null`?

Reply:
Yes, I like that solution more actually!